main.tf

```diff
@@ -37,6 +37,15 @@ resource "null_resource" "global_secondary_indexe_names" {
   triggers = "${map("name", lookup(var.global_secondary_index_map[count.index], "name"))}"
 }
 
+resource "null_resource" "local_secondary_indexe_names" {
```

Comment:
fix typo `local_secondary_index_names`

Reply:
Done (FWIW - it's mispelled on the GSI null resource above)

Reply:
can you please fix that as well?

Reply:
done